### PR TITLE
INTMDB-270: Added  'with_default_alerts_settings' to project resource

### PR DESCRIPTION
**Pull request by abner-dou**
## Description

A new property (`with_default_alerts_settings`) was added to `project` resource in order to disable default alerts on project creation. this property is set to `true` by default

Link to any related issue(s):[INTMDB-270](https://jira.mongodb.org/browse/INTMDB-270)

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have read the Terraform contribution guidelines
- [x] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

File: mongodbatlas/resource_mongodbatlas_project.go [hunk 1]
```diff
@@ -91,15 +97,15 @@ func resourceMongoDBAtlasProjectCreate(ctx context.Context, d *schema.ResourceDa
 
 	project, _, err := conn.Projects.Create(ctx, projectReq, createProjectOptions)
 	if err != nil {
-		return diag.FromErr(fmt.Errorf(errorProjectCreate, err))
```
Comment (themantissa) [inline mongodbatlas/resource_mongodbatlas_project.go:94]: This is not in the PR description, can you describe?  Is this updates for the newer SDK that weren't made yet?

File: website/docs/r/project.html.markdown [hunk 2]
```diff
@@ -19,6 +19,7 @@ resource "mongodbatlas_project" "test" {
   name   = "project-name"
   org_id = "<ORG_ID>"
   project_owner_id = "<OWNER_ACCOUNT_ID>"
+  with_default_alerts_settings = false
```
Comment (themantissa) [inline website/docs/r/project.html.markdown]: We don't want everyone doing this by default so please remove from the example. Those that need this simply need the description you have below.

**Review by themantissa — CHANGES_REQUESTED** [review]
Question and change. Thanks!

File: mongodbatlas/resource_mongodbatlas_project.go (hunk 1 above)
Comment (abner-dou) [reply mongodbatlas/resource_mongodbatlas_project.go:94]: I was working on this file and I noticed this, the correct use is `diag.Errorf(.....)` not `diag.FromErr(fmt.Errorf(....))` (this way works but is not correct)

File: website/docs/r/project.html.markdown (hunk 2 above)
Comment (abner-dou) [reply website/docs/r/project.html.markdown]: already changed :D

File: mongodbatlas/resource_mongodbatlas_project.go (hunk 1 above)
Comment (themantissa) [reply mongodbatlas/resource_mongodbatlas_project.go:94]: Glad to have it corrected but next time this kind of things come up please describe it and why in the PR description.  That way it's not a surprise. Thank you.

**Review by themantissa — APPROVED** [review]
LGTM, thanks!

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Review by thetonymaster — APPROVED** [review]
LGTM

**Pull request merged**